test/Common/DurableTaskEndToEndTests.cs

```diff
@@ -63,9 +63,11 @@ private void StartLogCapture()
         {
             if (this.useTestLogger)
             {
+                // Use GUID for eventsource, as TraceEventProviders.GetProviderGuidByName() is causing
+                // the CI to abort runs.

```

Comment:
Interesting, do you know why looking up EventSource events by name would lead to an aborted run? Or is this just from our experience so far?

Reply:
Not sure, but no one else has experience the stack trace. My guess is something to do with concurrency. I tried revving the dependency that code lives in to see if they have fixed the bug, but it happened once, so I am trying to lookup by GUID instead to see if that fixes the issue.